src/onboarding/Start.js

```diff
@@ -351,43 +374,140 @@ class StartContent extends React.PureComponent {
   }
 }
 
+const ResponsiveStartContent = props => (
+  <React.Fragment>
+    <BreakPoint to="medium">
+      <StartContent {...props} responsiveLanding />
+    </BreakPoint>
+    <BreakPoint from="medium">
+      <StartContent {...props} />
+    </BreakPoint>
+  </React.Fragment>
+)
+
+const DomainStatus = styled(Text)`
+  display: block;
+  margin: -10px 0 0 5px;
+`
+
+const SubmitWrap = styled.span`
+  height: 40px;
+  display: flex;
+
+  ${breakpoint(
+    'medium',
+    `
+      display: inline;
+      text-align: unset;
+    `
+  )}
+`
+
+const StyledSubmitButton = styled(Button)`
+  margin-left: auto;
+
+  ${breakpoint(
+    'medium',
+    `
+      margin-left: unset;
+    `
+  )}
+`
+
+const Warning = styled.div`
+  background: rgba(255, 195, 70, 0.09);
+  border-radius: 3px;
+  padding: 13px;
+  margin-bottom: 45px;
+  font-size: 15px;
+
+  & span {
+    font-weight: bold;
+  }
+`
+
 const Main = styled(animated.div)`
   display: flex;
   align-items: center;
   justify-content: flex-start;
   width: 100%;
   height: 100%;
-  padding: 100px;
+  padding: 24px;
+  background: url(${logo}) no-repeat top center;
+  background-size: calc(100vw - 32px);
+  background-position: 50% 16.666666vh;
+
+  ${breakpoint(
+    'medium',
+    `
+      padding: 100px;
+      background: none;
```

Comment:
I think on desktop we still want this background, but with a different `background-size` and `background-position`?

Reply:
Yes, but this background is part of the whole `Onboarding` background, I added it here (`Start`) only for the landing page.